examples/svg-interoperability/package.json

```diff
@@ -4,7 +4,11 @@
   "license": "MIT",
   "scripts": {
     "start": "webpack-dev-server --progress --hot --port 3000 --open",
```

Comment:
shall we remove all the _--port 3000_ to avoid conflicting ports when debugging multiple examples?
webpack-dev-server will assign one if not specified.